packages/renderless/src/common/deps/date.ts

```
@@ -230,7 +230,7 @@ fecha.format = (dateObj, mask, i18nSettings) => {
     dateObj = new Date(dateObj)
   }
 
-  if (!isDate(dateObj) || isNaN(dateObj.getTime())) {
+  if (!type.isDate(dateObj) || isNaN(dateObj.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)